newdemojail.yaml

```
@@ -0,0 +1,772 @@
+entityDefinitions:
+  inmate:
+    fqn: "general.person"
+    entitySetName: "DemoPerson"
```

```suggestion
    entitySetName: "DemoInmates"
```
There is already an existing entity set called DemoInmates. Do you plan on merging with it, or replacing it?

---

Merging for now and possibly ultimately replacing if we decide to delete old data